File: Python Tutorials/Python class Notes/prime.py
def prime(number):
    if number == 1:
        print("Universal Number")
    elif number > 1:
        for i in range(2, (number//2)+1):
            if number % i == 0:
                print(number, " is not prime")
                break
        else:
            print(number, " is prime")


number = int(input("Enter a number:"))
prime(number)

File: JAVA/Practice/Prime.java
import java.util.Scanner;

public class Prime {
    public static void main(String[] args) {
        int num, i, count = 0;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a number to check prime");
        num = sc.nextInt();
        for (i = 1; i <= num; i++) {
            if (num % i == 0) {
                count++;
            }
        }
        if (count == 2) {
            System.out.println(num + " " + "is a prime number");
        } else {
            System.out.println(num + " " + "is not a prime number");
        }
        sc.close();
    }
}
